OracleTuxedo/core/dockerfiles/12.1.3/Dockerfile

```diff
@@ -7,25 +7,25 @@
 #
 
 # Pull base image
-FROM oraclelinux
+FROM container-registry.oracle.com/java/serverjre
 
 MAINTAINER Todd Little <todd.little@oracle.com>
 
 # Core install doesn't include unzip or gcc, add them
 # Create the installation directory tree and user oracle with a password of oracle
-RUN yum -y install unzip gcc file; yum -y clean all && \
+RUN yum -y install unzip gcc file which; yum -y clean all && \
     groupadd -g 1000 oracle; useradd -b /home -m -g oracle -u 1000 -s /bin/bash oracle && \
     echo oracle:oracle | chpasswd; echo root:samplesvm | chpasswd
```

Comment:
Do not hard-code passwords in the `Dockerfile`. This line should be removed as there is no need for either the `oracle` or `root` users to have a password anyway.